src/components/InputTime/utils.ts

```diff
@@ -152,3 +152,34 @@ export const guessTimeFromString = (string: string) => {
   const time = getShortTimeString(hours, minutes);
   return hasValidUserInput ? { time, hours, minutes } : {};
 };
+
+const getTotalMinutes = (timeString?: string) => {
+  const [hours, minutes] = (timeString || ':').split(':');
+  const totalMinutes = parseInt(hours, 10) * 60 + parseInt(minutes, 10);
+  return !Number.isNaN(totalMinutes) ? totalMinutes : undefined;
+};
```

Comment:
Cleverly terse.

Reply:
I experienced a surge of unexpected pride.
I don't think anyone's ever said that about me before, in any context. :)